### DISCUSSION TITLE
The methods of the Image Card don't work?

### DISCUSSION THREAD

**Discussion by HueNa** [imfing/hextra#216] [Q&A] [answered]
Hello, 

I tried the image processing of the [Image Card](https://imfing.github.io/hextra/docs/guide/shortcodes/cards/#image-card),
but it seems nothing happened.

What should I do to make it work?
I'm afraid the options may be incorrect.
Or, the image processing doesn't work originally with the Image Card?

```
{{< cards >}}
  {{< card link="#" title="Crop1" image="docs/picture1.jpg" method="Crop" options="100x100 TopLeft" subtitle="Try to crop 100x100 TopLeft">}}
  {{< card link="#" title="Resize1" image="docs/picture1.jpg" method="Resize" options="100x100" subtitle="Try to resize 100x100">}}
  {{< card link="#" title="Fit1" image="docs/picture1.jpg" method="Fit" options="100x100" subtitle="Try to fit 100x100">}} 
  {{< card link="#" title="Fill1" image="docs/picture1.jpg" method="Fill" options="100x100" subtitle="Try to fill 100x100">}} 
  
  {{< card link="#" title="Crop2" image="docs/picture2.jpg" method="Crop" options="100x100 TopRight" subtitle="Try to crop 100x100 TopRight">}}
  {{< card link="#" title="Resize2" image="docs/picture2.jpg" method="Resize" options="200x200" subtitle="Try to resize 200x200">}}
  {{< card link="#" title="Fit2" image="docs/picture2.jpg" method="Fit" options="200x200" subtitle="Try to fit 200x200">}}
  {{< card link="#" title="Fill2" image="docs/picture2.jpg" method="Fill" options="200x200" subtitle="Try to fill 200x200">}}
{{< /cards >}}
```

Above code results in the following:
For picture1.jpg: 240 x 480,
![picture1_result](https://github.com/imfing/hextra/assets/22568913/e655169a-2d1b-406a-b6f6-97bc76e9225f)

For picture2.jpg: 640 x 480,
![picture2_result](https://github.com/imfing/hextra/assets/22568913/0dfb0c46-a07d-4e18-a4c4-71ab37967e66)

I also attached the modified hextra-starter-template to reproduce my situation.
[Test.zip](https://github.com/imfing/hextra/files/13422440/Test.zip)

Any help or hints will be appreciated.

**Comment by imfing** [comment, marked as answer]
The images need to be put under `assets/`

please see below first further details 
https://imfing.github.io/hextra/docs/guide/shortcodes/cards/#image-card

**Reply by HueNa** [reply]
Hello imfing,
Thank you for your reply.

I read the page you suggested again.
With your advice, I now understand the image process works only for the images under the `assets/`.

Thank you again, and sorry to bother you.
Best regards
